lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -71,14 +70,17 @@ function dosomething_rogue_send_reportback_to_rogue($values, $user = NULL) {
     'file' => isset($values['file']) ? $values['file'] : NULL,
     'caption' => isset($values['caption']) ? $values['caption'] : NULL,
     'status' => isset($values['status']) ? $values['status'] : 'pending',
-    'crop_x' => $values['crop_x'],
-    'crop_y' => $values['crop_y'],
-    'crop_width' => $values['crop_width'],
-    'crop_height' => $values['crop_height'],
-    'crop_rotate' => $values['crop_rotate'],
     'source' => isset($values['source']) ? $values['source'] : NULL,
   ];
 
+  if ($values['crop_x']) {
```

Comment:
@angaither I'm playing around with this and think it might make it a little confusing to pull out to a helper function to prepare `$data` to send since here, we'd have to send many different variables to the function (`$northstar_id`, `$user`, `$values`, and `$run`) while in `dosomethign_rogue.cron.inc`, it gets these values from the table/from the `$task` object. I think it makes more sense to keep as is unless I'm not understanding correctly?